examples/guides/gpio/Makefile

```diff
@@ -0,0 +1,29 @@
+# name of your application
+APPLICATION = hello-world
+
+# Change this to your board if you want to build for a different board
+BOARD ?= feather-nrf52840-sense
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../../..
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+DEVELHELP ?= 1
+
+# This board requires a start sleep to actually catch the printed output
+USEMODULE += shell
```

Comment:
???

Reply:
Leftover from another Makefile for a board oops :sweat_smile: